src/convertToObject.js

```diff
@@ -7,6 +7,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+
+  const x = sourceString
+    .toString()
+    .split(';')
+    .map((item) => item.toString().split(':'));
+
+  x.map((item) => {
+    if (!(item[1] === undefined)) {
+      result[item[0].trim()] = item[1].trim();
+    }
```

Comment:
[CODE PRACTICE] - Instead of using 'map' for side-effects, you should use 'forEach' when you intend to perform an operation for each item without creating a new array. The 'map' function is intended to transform each element and return a new array.